pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ValidationError
+from django.core.management import CommandError
+from django.core.validators import validate_email
+
+from pootle.core.delegate import formats
+from pootle_fs.presets import FS_PRESETS
+from pootle_fs.delegate import fs_plugins
+from pootle_language.models import Language
+from pootle_project.models import Project
+from pootle_project.models import PROJECT_CHECKERS
+
+
+class Command(BaseCommand):
+    help = "Creates a new project from command line."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'code',
+            action='store',
+            help='Project code',
+        )
+        parser.add_argument(
+            "--name",
+            action="store",
+            dest="name",
+            help="Project name",
+        )
+        parser.add_argument(
+            "--filetype",
+            action="append",
+            dest="filetypes",
+            choices=formats.get().keys(),
+            default=[],
+            help="File types: {0}".format(
+                 " | ".join(formats.get().keys())),
+        )
+        mapping_group = parser.add_mutually_exclusive_group(required=True)
+        mapping_group.add_argument(
+            "--preset-mapping",
+            action="store",
+            dest="preset_mapping",
+            choices=FS_PRESETS.keys(),
+            help="Preset filesystem layout: {0}".format(
+                 " | ".join(FS_PRESETS.keys())),
+        )
+        mapping_group.add_argument(
+            "--mapping",
+            action="store",
+            dest="mapping",
+            help="Custom mapping layout",
```

Comment:
`Custom filesystem layout`. Also I would include some help on how that layout must look like.